MiniTwit.FlagTool.Tests/MiniTwitTestContext.cs

```diff
@@ -0,0 +1,37 @@
+using System.Runtime.CompilerServices;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using MiniTwit.Entities;
+
+namespace MiniTwit.FlagTool.Tests
+{
+    public class MiniTwitTestContext : MiniTwitContext
+    {
+        private readonly SqliteConnection _connection;
+
+        public MiniTwitTestContext(DbContextOptions<MiniTwitContext> options, SqliteConnection connection) :
+            base(options)
+        {
+            _connection = connection;
+        }
+
+        public static MiniTwitTestContext CreateMiniTwitContext(
+            [CallerMemberName] string testName = "",
+            [CallerFilePath] string testNamePart2 = ""
+        )
+        {
+            var connection = new SqliteConnection("Datasource=:memory:");
+            connection.Open();
+            var options = new DbContextOptionsBuilder<MiniTwitContext>().UseSqlite(connection);
+            var context = new MiniTwitTestContext(options.Options, connection);
+            context.Database.EnsureCreated();
+            return context;
+        }
+
+        public override void Dispose()
+        {
+            base.Dispose();
+            _connection?.Dispose();
+        }
+    }
+}
```

Comment:
I really wish I had a need way to share this between projects. This is a duplicate class.

Reply:
IMO the context would fit well in the same package as the entities (even if that means renaming the entities package should be renamed).